shared/src/commonMain/kotlin/co/nimblehq/blisskmmic/domain/usecase/LogInUseCase.kt

```diff
@@ -3,15 +3,18 @@ package co.nimblehq.blisskmmic.domain.usecase
 import co.nimblehq.blisskmmic.domain.model.Token
 import co.nimblehq.blisskmmic.domain.repository.TokenRepository
 import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.map
 
 interface LogInUseCase {
 
     operator fun invoke(email: String, password: String): Flow<Token>
 }
 
-class LogInUseCaseImpl(private val repository: TokenRepository) : LogInUseCase {
+class LogInUseCaseImpl(
+    private val tokenRepository: TokenRepository
+    ) : LogInUseCase {
```

Comment:
```suggestion
class LogInUseCaseImpl(
    private val tokenRepository: TokenRepository
) : LogInUseCase {
```

Reply:
@minhnimble This is Android studio autocorrect, do you know which settings is related to this?

Reply:
Fixed. Seems to be because I need to add final `,`